### PR TITLE
Pass sentry info in android release builds

### PR DESCRIPTION
**Pull request by strseb**
We now have a vpn-sentry project. It's url / key are secret so let's get those from taskcluster :)

### CODE REVIEW

File: scripts/android/cmake.sh [hunk 1]
```diff
@@ -55,6 +55,16 @@ while [[ $# -gt 0 ]]; do
     RELEASE=
     shift
     ;;
+  --sentrydsn)
+    SENTRY_DSN="$2"
```
Comment (brizental) [inline scripts/android/cmake.sh:59]: What does DSN stand for?

**Review by brizental — APPROVED** [review]
:100:

File: scripts/android/cmake.sh (hunk 1 above)
Comment (strseb) [reply scripts/android/cmake.sh:59]: i do not know, sentry calls that value that :D

Comment (brizental) [reply scripts/android/cmake.sh:59]: LOL SHIP IT

Comment (brizental) [reply scripts/android/cmake.sh:59]: (It's Data Source Name [apparently](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjylOKjv8T7AhWQSPEDHSUBCzYQFnoECBIQAQ&url=https%3A%2F%2Fdocs.sentry.io%2Fproduct%2Fsentry-basics%2Fdsn-explainer%2F&usg=AOvVaw2dtdfVxf8bL3rzFraKUCR6))

**Pull request merged**